lib/util.rb

```diff
@@ -122,6 +122,17 @@ def self.days_in_range(data)
 
 # methods for gathering measurement data into hashes
 class Report
+  # this method returns an array of all the resources (without their
+  # measurements) a project has associated with it
+  def self.get_resources(project)
+    all_resources = []
+    plugin_matrix.each do |resource_type, _measurements|
+      all_resources.push(project.send("#{resource_type}_resources"))
```

Comment:
What happens here if the project doesn't have db resources, but db is in the plugin matrix?